openedx/core/djangoapps/user_api/accounts/__init__.py

```diff
@@ -61,7 +61,7 @@
     "It looks like {email_address} belongs to an existing account. "
     "Try again with a different email address."
 )
-AUTHN_EMAIL_CONFLICT_MSG = _("This email is already associated with an existing or previous edX account")
+AUTHN_EMAIL_CONFLICT_MSG = _("This email is already associated with an existing or previous {platform_name} account")
```

Comment:
Can we format the string here instead of doing it on the usages of this variable? Also, use f-string (`f''`) instead of format.

Reply:
@waheedahmed  I have addressed the comment.